### PR TITLE
Update Libevent

### PR DESCRIPTION
**Pull request by isaachier**
Seems to build fine: https://travis-ci.org/isaachier/hunter/builds/289562819.

### CODE REVIEW

File: cmake/projects/Libevent/hunter.cmake [hunk 1]
```diff
@@ -9,8 +9,8 @@ include(hunter_pick_scheme)
 hunter_add_version(
     PACKAGE_NAME Libevent
     VERSION "2.1.8"
-    URL "https://github.com/hunter-packages/libevent/archive/v2.1.8-p2.tar.gz"
-    SHA1 "492abb962dad3071f34bbe4975e28768fd29edf0")
+    URL "https://github.com/hunter-packages/libevent/archive/v2.1.8-p3.tar.gz"
```
Comment (ruslo) [inline cmake/projects/Libevent/hunter.cmake]: Add new `hunter_add_version` instead and update `default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/update.html

Comment (isaachier) [reply cmake/projects/Libevent/hunter.cmake]: OK will do

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2556
* https://travis-ci.org/ingenue/hunter/builds/289578841?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2560
* https://travis-ci.org/ingenue/hunter/builds/289656464?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.125

**Pull request merged**